app/models/timdex_search.rb

```diff
@@ -53,6 +53,7 @@ class TimdexSearch < TimdexBase
             matchedField
             matchedPhrases
           }
+          sourceLink
```

Comment:
Such a small change to result in such a mess of cassette updates :)

Reply:
Right? 🤦